cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst

```diff
@@ -22,9 +22,9 @@ Localization (l10n) is adapting a program to local language and cultural habits.
 
 Use the locale directory to provide internationalized strings for your XBlock project.
 For more information on how to enable translations, visit the
-`Open edX XBlock tutorial on Internationalization <https://edx.readthedocs.org/projects/xblock-tutorial/en/latest/edx_platform/edx_lms.html>`_.
+`Adding Translated Strings to your XBlock <https://edx.readthedocs.io/projects/xblock/en/latest/xblock-tutorial/edx_platform/edx_lms.html#adding-translated-strings-to-your-xblock>`_.
```

Comment:
This is only applicable for non-openedx XBlocks.

github.com/openedx XBlocks has full `atlas` support: 

And this link should be used instead:

 - https://docs.openedx.org/en/latest/developers/how-tos/enable-translations-new-repo.html

Reply:
You're right.